packages/spark-core/src/lib/spark-http-error.js

```diff
@@ -6,17 +6,43 @@
 
 import extendError from 'extend-error';
 import {HttpError} from '@ciscospark/http-core';
+import {isArray, pick} from 'lodash';
 
 const SparkHttpError = extendError(HttpError, {
   parseFn(res, ...rest) {
+    /* eslint complexity: [0] */
     /* eslint prefer-reflect: [0] */
     let message = HttpError.prototype.parseFn.call(this, res, ...rest);
 
+    let body = res.body;
```

Comment:
This parses just the description out of the error. HttpError parses the entire message. Initially I added the description parsing as part of HttpError, but you mentioned that it should be moved to SparkHttpError instead